test/jsonwp-proxy/proxy-specs.js

```diff
@@ -38,8 +38,8 @@ function mockProxy (opts = {}) {
 
 describe('proxy', function () {
   it('should override default params', function () {
-    let j = mockProxy({server: '127.0.0.2'});
-    j.server.should.equal('127.0.0.2');
+    let j = mockProxy({host: '127.0.0.2'});

```

Comment:
This is the wrong direction. `server` is the right one (https://github.com/appium/appium-base-driver/blob/fa8cfaafc3930c5f1fb5d14209f2f121563183ff/lib/jsonwp-proxy/proxy.js#L17)